### PR TITLE
Lists integration

### PR DESCRIPTION
**Pull request by kartikjagdale**
Create List Service

### CODE REVIEW

File: app/controllers/api/v0/lists_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Api::V0
+  class ListsController <  ApiController
+
+    def suggest
+      list = List.all.includes(:members);
+      render json: list.as_json(include: :members);
```
Comment (chandradot99) [inline app/controllers/api/v0/lists_controller.rb]: why are you including List members twice ??

File: app/services/twitter_api/create_list.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        config.consumer_secret     = ENV['TWITTER_SECRET']
+        config.access_token        = params[:access_token]
+        config.access_token_secret = params[:secret_token]
+      end
+
+      @db_list = list
+      @usernames =  @db_list.members.pluck(:screen_name)
+      @user = user
+      @result = { status: false }
+      return @result
```
Comment (ashish173) [inline app/services/twitter_api/create_list.rb]: user `super` here.

File: app/controllers/api/v0/lists_controller.rb (hunk 1 above)
Comment (ashish173) [reply app/controllers/api/v0/lists_controller.rb]: Lol semicolon @pkrawat1 😬

Comment (pkrawat1) [reply app/controllers/api/v0/lists_controller.rb]: they are allowed just like in javascript

**Review by pkrawat1 — COMMENTED** [review]
add test cases side by side. Try 1-2 test cases. This should be simple as its ruby.

File: app/controllers/api/v0/lists_controller.rb (hunk 1 above)
Comment (ashish173) [reply app/controllers/api/v0/lists_controller.rb]: @pkrawat1 I know 😛  made me laugh though

**Comment by ashish173** [comment]
@kartikjagdale: @pkrawat1 even I think so, that's a very good point.

**Comment by ashish173** [comment]
@kartikjagdale please merge this after you have made the changes,

**Pull request merged**